core/jvm/src/test/scala/fs2/Pipe2Spec.scala

```diff
@@ -150,17 +151,70 @@ class Pipe2Spec extends Fs2Spec {
 
     "interrupt (1)" in forAll { (s1: PureStream[Int]) =>
       val s = async.mutable.Semaphore[IO](0).unsafeRunSync()
-      val interrupt = Stream.emit(true) ++ Stream.eval_(s.increment)
+      val interrupt = mkScheduler.flatMap { _.sleep_[IO](50.millis) }.compile.drain.attempt
       // tests that termination is successful even if stream being interrupted is hung
       runLog { s1.get.covary[IO].evalMap(_ => s.decrement).interruptWhen(interrupt) } shouldBe Vector()
     }
 
     "interrupt (2)" in forAll { (s1: PureStream[Int]) =>
+      val s = async.mutable.Semaphore[IO](0).unsafeRunSync()
+      val interrupt = Stream.emit(true) ++ Stream.eval_(s.increment)
+      // tests that termination is successful even if stream being interrupted is hung
+      runLog { s1.get.covary[IO].evalMap(_ => s.decrement).interruptWhen(interrupt) } shouldBe Vector()
+    }
+
+    "interrupt (3)" in {
+      // tests the interruption of the constant stream
```

Comment:
Super minor style nitpick -- these comments can be either put in to the test name -- e.g. `intterupt (3) - interruption of constant stream` or wrapped in an `info("interruption of constant stream")`

Reply:
not sure about info sytax, xoun you just write quick example pls?

Reply:
```scala
"interrupt (3)" in {
  info("tests interruption of the constant stream")
  ...
}
```

It just prints the info string to the test output in test reports. In this case, it's probably better in the test name itself.

Reply:
I have recently experimented with that, however info() causes to print out the text at every `Gen` case, so perhaps 100x or so for every test. 
I am not sure about moving test cases to the names, not sure what that will do with tools that using names of tests to filter execution of only one single test case.

Reply:
Ouch. Moving text description to test name works fine -- e.g., `testOnly *MergeJoinSpec -- -z interrupt` will run all the tests that include the word `interrupt` in their name